packages/wallet-sdk/src/core/provider/method.ts

```diff
@@ -0,0 +1,44 @@
+const mapping = {
```

Comment:
consolidating all method categorization info,
used to be `canHandle` function on each "RequestHandler"